_common/GFMAlerts.php

```
@@ -0,0 +1,64 @@
+                    'warning' => '<svg class="octicon octicon-alert mr-2" viewBox="0 0 16 16" version="1.1" width="16" height="16" aria-hidden="true"><path fill="orange" d="M6.457 1.047c.659-1.234 2.427-1.234 3.086 0l6.082 11.378A1.75 1.75 0 0 1 14.082 15H1.918a1.75 1.75 0 0 1-1.543-2.575Zm1.763.707a.25.25 0 0 0-.44 0L1.698 13.132a.25.25 0 0 0 .22.368h12.164a.25.25 0 0 0 .22-.368Zm.53 3.996v2.5a.75.75 0 0 1-1.5 0v-2.5a.75.75 0 0 1 1.5 0ZM9 11a1 1 0 1 1-2 0 1 1 0 0 1 2 0Z"></path></svg>'
+                );
+                $iconType .= $icon[$alertType] . $alertTitle; // return string
+                $alertContent .= ltrim($matches[2]);
+    
+                return $block = array(
+                    'element' => array(
+                        'name' => 'div',
+                        'attributes' => array('class' => "markdown-alert markdown-alert-$alertType"),
+                        'handler' => 'elements',
+                        'text' => array(
+                            array(
+                                'name' => 'p',
+                                'attributes' => array('class' => "markdown-alert-title"),
+                                'rawHtml' => $iconType
+                            ),
+                            array(
+                                'name' => 'p',
+                                'text' => $alertContent,
+                            )
+                        )
+                    )
+                );
+            }
+            return parent::blockQuote($block);
+        }
+        catch (Error $e) {
+            return $e;
+        }
```

I don't think it is correct to return $e? It would be safer to eliminate the `try`/`catch` altogether and let errors bubble through